client/ayon_core/tools/sceneinventory/models/containers.py

```diff
@@ -222,61 +227,70 @@ def get_container_items_by_id(self, item_ids):
     def get_representation_info_items(self, representation_ids):
         output = {}
         missing_repre_ids = set()
+        missing_repre_ids_by_project = {}
+        containers = self._controller.get_containers()
+
         for repre_id in representation_ids:
             try:
                 uuid.UUID(repre_id)
             except ValueError:
                 output[repre_id] = RepresentationInfo.new_invalid()
                 continue
 
+            project_name = self._find_project_name(containers, repre_id)
+            if project_name is None:
+                project_name = self._controller.get_current_project_name()
+
             repre_info = self._repre_info_by_id.get(repre_id)
             if repre_info is None:
                 missing_repre_ids.add(repre_id)
+                missing_repre_ids_by_project.update({project_name: repre_id})
```

Comment:
```suggestion
                missing_repre_ids_by_project.setdefault(
                    project_name, set()
                ).add(repre_id)
```

Reply:
This could be much prettier when just using a `defaultdict(set)`?